doc/user/content/sql/drop-index.md

```diff
@@ -42,8 +45,12 @@ If you drop the primary index and want to recreate it:
 Alternatively, you can also [`CREATE INDEX`](../create-index) to materialize any
 view.
 
+Dropping the only index on a materialized view does not remove the view entirely; it merely transforms it to a non-materialized view.
+
 ## Examples
 
+### Drop indexes (no dependencies)
```

Comment:
Maybe `Drop indexes without dependent objects` and `Drop indexes with dependent objects` below to make these more symmetric?